visualizer/src/App.cpp

```diff
@@ -7,36 +7,61 @@
 using namespace mode7;
 
 static Shader test_shader;
-static Scene test_mesh;
+static Scene test_monk;
 static Scene test_cube;
-static float x;
-
+static float angle;
+static float radius; 
+static float moved; 
+static float x, y, z; 
 void App::init()
 {
     test_shader.open("assets/shaders/test_v.glsl", "assets/shaders/test_f.glsl");
-    test_mesh = ModelLoader::open("assets/models/suzanne.dae");
-    test_cube = ModelLoader::open("assets/models/test_cube.dae");
+    test_monk = ModelLoader::open("assets/models/suzanne.dae"); //Monkey object
+    //test_cube = ModelLoader::open("assets/models/test_cube.dae"); //Cube object
 
-    test_cube.position.z = -3.f;
+    test_monk.rotate(0.0f, 0.0f, 0.0f); //Rotate the object model to its front
```

Comment:
the .rotate() method does not set rotation, but applies a rotation. Calling .rotate(0,0,0) makes no change to the orientation of an object

Reply:
The line has been removed.